shell/config/product/explorer.js

```diff
@@ -192,7 +192,30 @@ export function init(store) {
   headers(WORKLOAD_TYPES.DAEMON_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.REPLICA_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.STATEFUL_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
-  headers(WORKLOAD_TYPES.JOB, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Completions', 'Duration', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
+  headers(WORKLOAD_TYPES.JOB, [
+    STATE,
+    NAME_COL,
+    NAMESPACE_COL,
+    WORKLOAD_IMAGES,
+    WORKLOAD_ENDPOINTS,
+    'Completions',
+    {
+      name:      'duration',
+      labelKey:  'tableHeaders.duration',
+      value:     (row) => {
+        const getValue = store.getters['type-map/rowValueGetter'](
+          store.getters['cluster/schemaFor'](WORKLOAD_TYPES.JOB),
```

Comment:
I am not sure about the full implications, but a user might not have access to a certain schema. We usually use the `canList` getter https://github.com/rancher/dashboard/blob/master/shell/plugins/dashboard-store/getters.js#L199 to check first...
Ex: not the same store, but works the same... https://github.com/rancher/dashboard/blob/master/shell/list/provisioning.cattle.io.cluster.vue#L42

Maybe it's wise to add it here and prepare for a fallback.

Reply:
Let's leave this for now as a user without permissions to access JOBS won't even be allowed to click on the list view link to access it, so therefore this possible scenario might not even manifest itself.